### PR TITLE
Refactor: Delete store and unused code

### PR DESCRIPTION
**Pull request by BastinJafari**

### CODE REVIEW

**Comment by Roshanjossey** [comment]
image

Percy caught something. @BastinJafari, could you check the elements in the footer? Not sure what it has to do with store though.

**Comment by BastinJafari** [comment]
I deleted a lot of components that I thought wouldn't get used anymore. 
Regarding this display bug, I can't reproduce it.

File: src/index.tsx [hunk 1]
```diff
@@ -8,22 +8,13 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./assets/main.scss";
 import "./App.css";
 import store from "./redux";
-
-let App;
```
Comment (Roshanjossey) [inline src/index.tsx:12]: I think we still need this. From what I understand, the usecase is to deploy standalone Hub. @imsergiy, could you confirm?

File: src/layouts/index.ts [hunk 2]
```diff
@@ -1,4 +1,3 @@
 import IconSidebar from "./IconSidebar";
-import HubLayout from "./HubLayout";
 
-export { IconSidebar,HubLayout };
+export { IconSidebar };
```
Comment (Roshanjossey) [inline src/layouts/index.ts:3]: Same here.

**Review by Roshanjossey — CHANGES_REQUESTED** [review]
I think we shouldn't remove Hub routes and code associated with it. From what I understand, that use case is still needed

**Review by Roshanjossey — APPROVED** [review]

**Pull request merged**